src/fmu/sumo/explorer/objects/table.py

```diff
@@ -17,45 +18,48 @@ def __init__(self, sumo: SumoClient, metadata: dict) -> None:
             metadata: (dict): child object metadata
         """
         super().__init__(sumo, metadata)
-        self._dataframe = None
+        self._topandas = None
```

Comment:
Could maybe keep the name `_dataframe` for the internal name (as it stores a `pd.DataFrame`). `topandas` sounds like a method name to me

Reply:
@ovejorgensen here dd7f2f1eae9aff056e80e8d2371c7a4cece5befd